apps/frontend/components/Report/DroughtReport/DroughtReport.tsx

```diff
@@ -0,0 +1,21 @@
+import { DroughtDto } from '@wfp-dmp/interfaces';
+import { useMemo } from 'react';
+
+import { generateDroughtDetailedReport } from 'utils/aggregate/drought/generateDroughtDetailedReport';
+import { formatDroughtFields } from 'utils/formatRawToForm';
+
+export const DroughtReport = ({ forms }: { forms: DroughtDto[] }) => {
+  const detailedReport = useMemo(
+    () =>
+      generateDroughtDetailedReport(
+        forms.map(form => formatDroughtFields(form)),
+      ),
+    [forms],
+  );
+
+  return (
+    <>
+      <div>{JSON.stringify(detailedReport)}</div>
+    </>
+  );
+};
```

Comment:
Are we gonna be showing multiple report types on the same page?
If not, I feel like some of these files are unnecessarily duplicative.

Eg. DroughtReport.tsx and FloodReport.tsx share 90% of their structure